rtw_excel_report/models/models.py

```diff
@@ -997,6 +1001,11 @@ def _compute_sale_order_price_subtotal(self):
         for line in self:
             line.sale_order_price_subtotal = self.add_money_comma(line.price_subtotal) if line.price_subtotal else ''
     
+    @api.onchange('product_uom_qty', 'discount', 'price_unit')        
+    def _compute_sale_order_amount_no_rate(self):
+        for line in self:
+            line.sale_order_amount_no_rate = self.add_money_comma(line.product_uom_qty *line.price_unit) if (line.product_uom_qty or line.price_unit) else ''
```

Comment:
1. phần tính toán "sale_order_amount_no_rate" không thấy sử dụng "discount" thì sao lại cần thêm vào onchange()
2. line.product_uom_qty *line.price_unit -> format lại, trước và sau dấu "*" phải có space
3. điều kiện trong if có sử dụng or, nếu 1 trong 2 thằng None hoặc empty thì sao?
type của product_uom_qty và price_unit là gì? lưu type của "sale_order_amount_no_rate" là char filed có ổn ko?